data/abhipatel.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "Abhi Patel",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "MERN Stack web developer • **Open Source** contributor • Freelancer • WEB3.0 + Cloud Enthusiast",
+  "tags": ["Open Source", "Freelancer", "React.js", "Next.js", "Node.js"],
+  "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/abhirk247AbhiPatel0001"
+    },
+    { "icon": "FaGithub", "url": "https://github.com/AbhiPatel10" },
+    {
+      "icon": "FaLinkedin",
+      "url": "https://www.linkedin.com/in/abhipatel001/"
+    },
+    { "icon": "website", "url": "https://abhipatelportfolio.netlify.app/" }
+  ],
+  "avatar": "https://github.com/AbhiPatel10.png",
+  "links": [
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/abhipatel001/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AbhiPatel0001",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/AbhiPatel10",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Visit my personal website",
+      "url": "https://abhipatelportfolio.netlify.app/",
+      "icon": "website"
```

```suggestion
      "icon": "FaGlobe"
```
I believe there is no `website` icon I'd suggest using the globe icon

---

Done, I have made the changes suggested by you; please check and let me know if I need to change anything.